### PR TITLE
create the base for the project

### PR DESCRIPTION
**Pull request by Raymundogs**

### CODE REVIEW

**Review by danielaSantaCruz — APPROVED** [review]

File: src/main/java/core/api/ApiManager.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core.api;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
```
Comment (JessickaMoya) [inline src/main/java/core/api/ApiManager.java]: Delete blank line

File: src/main/java/core/api/ApiRequest.java [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core.api;
+
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+
```
Comment (JessickaMoya) [inline src/main/java/core/api/ApiRequest.java]: Delete blank line

File: src/main/java/core/selenium/WebElementAction.java [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+package core.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
```
Comment (JessickaMoya) [inline src/main/java/core/selenium/WebElementAction.java:17]: Delete blank lines between imports

File: src/main/java/core/api/ApiManager.java (hunk 1 above)
Comment (Raymundogs) [reply src/main/java/core/api/ApiManager.java]: solved it.

File: src/main/java/core/selenium/WebElementAction.java (hunk 3 above)
Comment (Raymundogs) [reply src/main/java/core/selenium/WebElementAction.java:17]: solved it.

File: src/main/java/core/api/ApiRequest.java (hunk 2 above)
Comment (Raymundogs) [reply src/main/java/core/api/ApiRequest.java]: solved it.

**Review by JessickaMoya — APPROVED** [review]

**Review by Paola-Xaq — APPROVED** [review]

**Pull request merged**